MotionMark/tests/resources/main.js

```diff
@@ -223,28 +226,32 @@ Controller = Utilities.createClass(
 
     _processControllerSamples: function()
     {
+        const processedSampleFrameLengthIndex = 2;
+        const processedSampleSmoothedFrameLengthIndex = 3;
```

Comment:
I think this set should include two `processedSample` indices: one for the time and the second for the complexity.